app/models/reports/region_summary_schema.rb

```diff
@@ -226,10 +226,16 @@ def visited_without_bp_taken(with_ltfu: false)
       values_at(field)
     end
 
-    def visited_without_bs_taken
+    memoize def visited_without_bs_taken
```

Comment:
Why is this `memoize` required?

Reply:
CMIIW I thought we were supposed to cache all the functions pulling data from tables.

Reply:
This `memoize` caches return values of methods at the object level (unrelated to caching to redis). We usually add this when the same method is being called multiple times in an object's lifetime, to avoid repeating computation. It comes from this gem https://github.com/tycooon/memery

Reply:
This method can get called multiple times in a  given request, which is why we've added the memoize keyword